core-web/libs/sdk/experiments/src/lib/persistence/index-db-database-handler.ts

```diff
@@ -0,0 +1,144 @@
+interface DbConfig {
+    db_name: string;
+    db_store: string;
+    db_key_path: string;
+}
+
+const DB_DEFAULT_VERSION = 1;
```

Comment:
use. constant `EXPERIMENT_DB_DEFAULT_VERSION ` here ?

Reply:
All the class, if you check, I didn't mention Experiments. I remove it on purpose.